src/Cookie/CookieJar.php

```diff
@@ -89,10 +89,10 @@ public static function shouldPersist(
     /**
      * Finds and returns the cookie based on the name
      *
-     * @param string $name cookie name to search for
+     * @param string $name cookie name to search for (null by default)
      * @return SetCookie|null cookie that was found or null if not found
      */
-    public function getCookieByName($name)
+    public function getCookieByName($name = null)
```

Comment:
Not sure if this is what @Nyholm pointed out. The problem is that when name is null, strcasecmp will fail on HHVM.